tests/e2e-insights-runtime-extractor-soak/soak-test/soak-test.sh

```
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -uo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+iteration=0
+while [ $iteration -le 120 ]; do
+  "${SCRIPT_DIR}/extract.sh"
+  sleep 30
+  echo "iteration: ${iteration}"
+  iteration=$(($iteration+1))
```

Shellcheck (0.11.0)

[style] $/${} is unnecessary on arithmetic variables.

(SC2004)